### PR TITLE
Mention autoscheduling in the Repair services

### LINKED ISSUE #352: Document Reaper's auto scheduling feature
**Is your feature request related to a problem? Please describe.**
We added initial support for auto scheduling in #224 but it is not covered in the docs.

**Describe the solution you'd like**
Provide an explanation of auto scheduling which should include what types of repair schedules will get created when it is enabled.

### PR DESCRIPTION
**Pull request by burmanm**
**What this PR does**:
Adds a small note to the access Repair services about autoscheduling.

**Which issue(s) this PR fixes**:
Fixes #352 

**Checklist**
- [ ] Changes manually tested
- [ ] Chart versions updated (if necessary)
- [ ] Automated Tests added/updated
- [x] Documentation added/updated
- [x] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/docs/topics/accessing-services/repair/_index.md [hunk 1]
```diff
@@ -88,6 +88,12 @@ Notice the new repair added to the list.
 
 See [Schedule a cluster repair](http://cassandra-reaper.io/docs/usage/schedule/).
 
+### Autoscheduling
+
+It is possible to automatically schedule repairs for all non-system keyspaces in the cluster. Cluster's keyspaces are monitored and any modification (adding or removing) are  detected. Adding new ones will get a schedule created for them and removing the keyspace will instead remove the corresponding repair schedule.
```
Comment (jsanda) [inline docs/content/en/docs/topics/accessing-services/repair/_index.md:93]: One minor nit. I think `Cluster's` should be `Clusters'`.

**Review by jsanda — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/topics/accessing-services/repair/_index.md (hunk 1 above)
Comment (burmanm) [reply docs/content/en/docs/topics/accessing-services/repair/_index.md:93]: Right, if multiple clusters are managed by one Reaper.

Comment (jdonenine) [reply docs/content/en/docs/topics/accessing-services/repair/_index.md:93]: How about switching to `With auto-scheduling, it is possible to automatically schedule repaors for all non-system keyspaces in a cluster.  A cluster's keyspaces are monitored and any modifications (additions or removals) are detected.  When a new keyspace is created, a new repair scheduled will be automatically created for that keyspace.  Conversely, when a keyspace is removed, the corresponding repair scheduled will be deleted.`

**Review by jdonenine — CHANGES_REQUESTED** [review]

**Comment by johnsmartco** [comment]
Good info, thanks guys.  Closing this PR without merging; superseded by non-conflicting edits in https://github.com/k8ssandra/k8ssandra/pull/460.

**Pull request closed without merging**